hackforla/peopledepot
# Setup mkdocs

- Fixes #166

### What changes did you make?

- created local server setup for MkDocs using docker image
- set up gh action to deploy docs site
- added documentation for MkDocs

### Why did you make the changes (we will use this info to test)?

- the local docker setup provides a way for devs to work on docs locally
- once merged, the updated docs will deploy automatically
- Documentation makes it easy to onboard new devs to use this

### How to test

- Check Actions tab your repo fork if you pulled and pushed this PR branch to it. You may have to set your gh-pages to deploy from a branch and set it to the gh-pages branch. After that, you should be able to view the docs at https://yourusername.github.io/peopledepot/. For example: https://fyliu.github.io/peopledepot/
- For using the local mkdocs server, see the documentation.
